pages/profile/[username].js

```diff
@@ -164,6 +164,7 @@ function Profile() {
                                         src={url}
                                         className="profile-image"
                                         shadow="4"
+                                        alt="User's Profile Image"
```

Comment:
We can make this even better in this way. 

```
alt={`${userRequest.user.name}'s Profile Pic`}
```

This will render the user's name in the image's alternative text, which seems clearer.